lib/app.js

```diff
@@ -1,17 +1,23 @@
 // import React/Redux libraries
 import React, { Component } from 'react'
 // import Bootstrap Grid components for layout
-import { Navbar, Grid, Row, Col } from 'react-bootstrap'
+import { Nav, Navbar, Grid, Row, Col } from 'react-bootstrap'
+
 // import OTP-RR components
 import {
   DefaultMainPanel,
+  DesktopNav,
   LineItinerary,
   Map,
   MobileMain,
+  NavLoginButtonAuth0,
   ResponsiveWebapp,
-  AppMenu
+  AppMenu,
+  otpUtils
 } from 'otp-react-redux'
 
+const { accountLinks } = otpUtils.auth
+
```

Comment:
`NavLoginButtonAuth0`, `AppMenu`, `accountLinks` (and therefore `otpUtils`) not used in this file.